src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -531,4 +531,12 @@ public function itsStateShouldBe($state)
             'The order state should be %2$s, but it is %s.'
         );
     }
+
+    /**
+     * @Then it should have a :state state
+     */
+    public function itShouldHaveState($state)
+    {
+        $this->indexPage->isSingleResourceOnPage(['state' => $state]);
+    }
```

Comment:
Not a good step, because it assumes that there is only one order in the list. That being said, we can change it in further PRs which check this and other order states...